frontend/src/component/layout/MainLayout/DraftBanner/DraftBanner.tsx

```diff
@@ -6,6 +6,7 @@ import { usePendingChangeRequests } from 'hooks/api/getters/usePendingChangeRequ
 import type { ChangeRequestType } from 'component/changeRequest/changeRequest.types';
 import { changesCount } from 'component/changeRequest/changesCount';
 import { Sticky } from 'component/common/Sticky/Sticky';
+import { useUiFlag } from '../../../../hooks/useUiFlag';
```

Comment:
Sorry for leaving this comment, but..

Reply:
actually I had it fixed before the comment but forgot to push :D